packages/netlify-cms-core/src/actions/config.js

```diff
@@ -111,17 +134,20 @@ export function applyDefaults(config) {
                 file = setDefaultPublicFolder(file);
                 file = file.set(
                   'fields',
-                  traverseFields(file.get('fields'), setDefaultPublicFolder),
+                  traverseFields(file.get('fields'), setDefaultPublicFolder, setSnakeCaseConfig),
                 );
                 return file;
               }),
             );
           }
 
-          if (!collection.has('sortableFields')) {
+          if (!collection.has('sortable_fields') && !collection.has('sortableFields')) {
             const backend = resolveBackend(config);
             const defaultSortable = selectDefaultSortableFields(collection, backend);
-            collection = collection.set('sortableFields', fromJS(defaultSortable));
+            collection = collection.set('sortable_fields', fromJS(defaultSortable));
+          } else if (collection.has('sortableFields')) {
+            collection = collection.set('sortable_fields', collection.get('sortableFields'));
+            delete collection['sortableFields'];
```

Comment:
This won't work since `collection` it an [Immutable](https://immutable-js.github.io/immutable-js/) instance.

Reply:
Okay, then I would leave that value as it is since as you mentioned before, users might access it in some other ways

Reply:
Good point(s). That part of the config is not accessible to custom user code (like widget configurations)  so you can safely delete `sortableFields` and only keep `sortable_fields`